builtin/mainmenu/tab_about.lua

```diff
@@ -119,11 +119,16 @@ return {
 			buildCreditList(previous_contributors) .. "," ..
 			";1]"
 
+		-- Render information
+		fs = fs .. "label[0.75,4.9;" ..
+			fgettext("Active render:") .. "\n" ..
```

Comment:
Apparently I marked this as resolved but I'm still suggesting this:
```suggestion
			fgettext("Active renderer:") .. "\n" ..
```

Reply:
It's likely that I accidentally overwrote this in the force push. Thanks for re-checking.